descriptors/DeploymentDescriptor-template.json

```diff
@@ -1,7 +1,7 @@
 {
-  "srvcId": "${artifactId}-${version}",
+  "srvcId": "@artifactId@-@version@",
   "nodeId": "localhost",
   "descriptor": {
-    "exec": "java -Dport=%p -jar ../${artifactId}/target/${artifactId}-fat.jar -Dhttp.port=%p embed_postgres=true"
+    "exec": "java -Dport=%p -jar ../@artifactId@/target/@artifactId@-@version@.jar"
```

Comment:
Is it related to some format change? I haven't seen such format before.

Reply:
Since we have `artifactId` and `version` duplicates in <parent> section in pom.xml, `maven-resources-plugin` filtering doesn't work for `${param}` syntax.